src/lib/selectors.ts

```diff
@@ -77,3 +79,37 @@ export const selectSinglePathDenomTraces = (
   }
   return query.data.filter(hasOnePath);
 };
+
+// Helper function to convert Duration to seconds string
+const durationToSeconds = (duration?: Duration): string => {
+  if (!duration?.seconds) return "0";
+  return duration.seconds.toString();
+};
+
+// Selector for Gov v1 parameters - converts API response to form data (ALL parameters from real chains)
+export const selectGovV1Params = (
+  query: UseQueryResult<GovV1Params, unknown>,
+): GovV1ParamFormData | undefined => {
+  if (!query?.data) return undefined;
+  
+  const params = query.data;
+  
+  // Use actual values from API, no arbitrary defaults - include ALL parameters that real chains return
+  return {
+    minDeposit: params.minDeposit,
+    maxDepositPeriod: durationToSeconds(params.maxDepositPeriod),
+    votingPeriod: durationToSeconds(params.votingPeriod),
```

Comment:
The `api` returns the params in snake case

```suggestion
    maxDepositPeriod: durationToSeconds(params.max_deposit_period),
    votingPeriod: durationToSeconds(params.voting_period),
```